src/benchmarks/micro/corefx/System.Runtime.Extensions/Perf.StreamWriter.cs

```diff
@@ -31,14 +30,12 @@ public IEnumerable<object> WriteLengthMemberData()
         [GlobalSetup]
         public void Setup()
         {
-            _memoryStream = new MemoryStream(MemoryStreamSize);
-            _streamWriter = new StreamWriter(_memoryStream, new UTF8Encoding(false, true), DefaultStreamWriterBufferSize, leaveOpen: true);
+            _streamWriter = new StreamWriter(Stream.Null, new UTF8Encoding(false, true), DefaultStreamWriterBufferSize, leaveOpen: true);
```

Comment:
Because the test is meant to measure the cost of StreamWriter, not the overheads of MemoryStream.  The test is conflating the two.  In contrast, Stream.Null has the minimal amount of overhead possible in any stream implementation.